add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -59,6 +59,16 @@ const ROOM_DELETE = {
   DELETE_FAIL: 1
 };
 
+/**
+ * Values that we segment sharing screen pause/ resume action telemetry probes into.
+ *
+ * @type {{PAUSED: Number, RESUME: Number}}
+ */
+const SHARING_SCREEN = {
+  PAUSED: 0,
+  RESUME: 1
```

Comment:
I didn't see this before, but you're stating 'Paused' in the passed tense (i.e. it already happened) and 'Resume' in present tense (i.e. it is happening right now). Can you make them both passed tense - s/resume/resumed/? (I think that'd make most sense for a counter metric.)